app/lib/pdf_filler/schedule_nj_hcc_pdf.rb

```diff
@@ -0,0 +1,36 @@
+module PdfFiller
+  class ScheduleNjHccPdf
+    include PdfHelper
+    include StateFile::NjPdfHelper
+
+    def source_pdf_name
+      "schedule-njhcc"
+    end
+
+    def initialize(submission)
+      @submission = submission
+
+      builder = StateFile::StateInformationService.submission_builder_class(:nj)
+      @parent_xml_doc = builder.new(submission).document
+    end
+
+    def hash_for_pdf
+      {
+        'Names as shown on Form NJ1040': get_name(@parent_xml_doc),
+        'Social Security Number': get_taxpayer_ssn,
+        Group1: "Choice1"
```

Comment:
Does `Group1` need to be in quotes too? [PEBBLE]
```suggestion
        'Group1': "Choice1"
```

Reply:
It actually gives me a Rubocop style error to use `'Group1'` with quotes
![image](https://github.com/user-attachments/assets/1e9407b5-2916-440d-9643-e76e62006384)